service/src/main/java/com/commercetools/pspadapter/payone/mapping/PayoneRequestFactory.java

```diff
@@ -33,4 +36,31 @@ public CaptureRequest createCaptureRequest(final PaymentWithCartLike paymentWith
         throw new UnsupportedOperationException("this request type is not supported by this payment method.");
     }
 
+    protected void mapFormPaymentWithCartLike(final AuthorizationRequest request,
+                                              final PaymentWithCartLike paymentWithCartLike,
+                                              final Logger logger) {
+        final Payment ctPayment = paymentWithCartLike.getPayment();
+        final CartLike ctCartLike = paymentWithCartLike.getCartLike();
+
+        MappingUtil.mapCustomFieldsFromPayment(request, ctPayment.getCustom());
+
+
+        try {
+            MappingUtil.mapCustomerToRequest(request, ctPayment.getCustomer());
+        } catch (final IllegalArgumentException ex) {
+            logger.debug("Could not fully map payment with ID {} {}", paymentWithCartLike.getPayment().getId(), ex.getMessage());
+        }
+
+        try {
+            MappingUtil.mapBillingAddressToRequest(request, ctCartLike.getBillingAddress());
+        } catch (final IllegalArgumentException ex) {
+            logger.error("Could not fully map payment with ID {} {}", paymentWithCartLike.getPayment().getId(), ex.getMessage());
```

Comment:
why these 3 mappings have different log levels (_debug_ for `customer` and `shippingAddress` and _error_ for `billingAddres`), when messages are absolutely the same. Looks like the function may be re-factored to use single try..catch for all three calls.